servers.json

```diff
@@ -8,6 +8,14 @@
             "funsmp.com"
         ]
     },
+    {
+        "id": "juvymc",
```

Comment:
Make sure your server logo is at least 512px by 512px

Reply:
Ok i edited it now do i have to make another pr or what idk how to update it